deploy/all-in-one.yaml

```diff
@@ -621,6 +621,224 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
```

Comment:
Did we manually test the all in one?
I guess the CI does not cover it yet..., in fact it should update it before release to cover it.